print_docs.py

```diff
@@ -157,10 +157,38 @@ def parse_bib_file(fname):
 if cl_args.w:
   site_root = cl_args.w
 
-mathlib_github_src_root = "{0}/blob/{1}/src/".format(mathlib_github_root, mathlib_commit)
+# contains an array of [prefix, replacement] strings to be rewritten by nav.js
+url_rewrites = []
```

Comment:
I'm storing the rewrites in an array like this since it might make it easier for `doc-gen` to be used by projects outside mathlib.